<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandIfCondition</class>
 <widget class="QDialog" name="CommandIfCondition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="m_pConditionTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>398</width>
     <height>432</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="m_pComparisonWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="m_pAND">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AND</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_pOR">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Condition Comparison:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Conditions:</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="m_pButtons">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pAddCondition">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Condition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pDeleteCondition">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Condition</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pButtons</sender>
   <signal>accepted()</signal>
   <receiver>CommandIfCondition</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtons</sender>
   <signal>rejected()</signal>
   <receiver>CommandIfCondition</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
